o8g/Sets/7a1d7bfd-db29-4812-8de1-01cf7ca73323/set.xml

```diff
@@ -0,0 +1,2813 @@
+      <property name="Political Skill" value="1" />
+      <property name="Glory" value="1" />
+      <property name="Deck" value="Dynasty" />
+      <property name="Unique" value="False" />
+    </card>
+    <card name="Hida Kisada" id="c7725dcb-9108-4067-9372-ad9e363c0415" size="dynasty">
+      <property name="Clan" value="Crab" />
+      <property name="Text">While you have not lost a conflict this phase, cancel the effects of the first action ability your opponent triggers from a card during each conflict.</property>
+      <property name="Type" value="Character" />
+      <property name="Traits" value="Bushi. Champion" />
+      <property name="Card Number" value="37" />
+      <property name="Cost" value="5" />
+      <property name="Military Skill" value="7" />
+      <property name="Political Skill" value="2" />
+      <property name="Glory" value="0" />
+      <property name="Deck" value="Dynasty" />
+      <property name="Unique" value="True" />
+    </card>
+    <card name="Hida Tomonatsu" id="b9591ff0-c32f-4bd0-9ead-88392c7e5161" size="dynasty">
+      <property name="Clan" value="Crab" />
+      <property name="Text">
+        <b>Reaction:</b> After this character wins a conflict as a defender, sacrifice it. Choose a non-unique attacking character – return that character to the top of its owner's deck.</property>
+      <property name="Type" value="Character" />
+      <property name="Traits" value="Bushi" />
+      <property name="Card Number" value="34" />
+      <property name="Cost" value="3" />
+      <property name="Military Skill" value="3" />
+      <property name="Political Skill" value="2" />
+      <property name="Glory" value="2" />
+      <property name="Deck" value="Dynasty" />
+      <property name="Unique" value="True" />
+    </card>
+    <card name="Hiruma Ambusher" id="8b4ece45-02c7-42a2-978b-d8ea103890d5" size="conflict">
+      <property name="Clan" value="Crab" />
+      <property name="Text">
+        <b>Reaction:</b> After this character enters play as a defender, choose a character – that character's abilities cannot be triggered during this conflict.</property>
+      <property name="Type" value="Character" />
+      <property name="Traits" value="Bushi. Scout" />
+      <property name="Card Number" value="130" />
+      <property name="Cost" value="2" />
+      <property name="Military Skill" value="2" />
+      <property name="Political Skill" value="2" />
+      <property name="Glory" value="0" />
+      <property name="Deck" value="Conflict" />
+      <property name="Unique" value="False" />
+      <property name="Influence Value" value="1" />
+    </card>
+    <card name="Hiruma Yōjimbō" id="59fee928-8b62-424e-8024-a31bbd620bd0" size="dynasty">
+      <property name="Clan" value="Crab" />
+      <property name="Text">This character cannot be declared as an attacker.</property>
+      <property name="Type" value="Character" />
+      <property name="Traits" value="Bushi. Yojimbo" />
```

Comment:
Not a big deal, but I noticed utf-8 characters are preserved for name but not traits (Yōjimbō vs Yojimbo)